daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
Yeah there is that of course. So then curses wise there should be no difference between linux and mac in the cmake files, except for the default setting,